src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -1019,76 +1002,167 @@ private AdsDataProvider() {
   }
 
   @DataProvider
-  public static Object[][] adsFloatingMedrec() {
+  public static Object[][] adsSlotSizeOasis() {
     return new Object[][]{
         {
-            "adtest",
-            "SyntheticTests/Oasis/FloatingMedrecOnLongPage",
-            223909212,
-            300,
-            250
+            new Page("adtest", "SyntheticTests/INCONTENT_LEADERBOARD/728x90"),
+            "InstantGlobals.wgAdDriverIncontentLeaderboardSlotCountries=[XX]",
+            ImmutableMap.<String, Object>builder()
+                .put("slotName", AdsContent.INCONTENT_LEADERBOARD)
+                .put("slotSize", new Dimension(728, 90))
+                .put("lineItemId", 239961972)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "lineItemId" 16 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)